opensearch_py_ml/ml_commons/rag_pipeline/rag/ingest.py

```diff
@@ -0,0 +1,194 @@
+# ingest_class.py
+
+import os
+import glob
+import json
+import tiktoken
+from tqdm import tqdm
+from colorama import Fore, Style, init
+from typing import List, Dict
+import csv
+import PyPDF2
+import boto3
+import botocore
+import time
+import random
+
+
+from opensearch_class import OpenSearchClass
+
+init(autoreset=True)  # Initialize colorama
+
+class IngestClass:
+    EMBEDDING_MODEL_ID = 'amazon.titan-embed-text-v1'
```

Comment:
This is a little bit confusing typically model ids look something like `xcnzLJMBbGw3ep23emly` and this describes a specific type of model maybe we ought to rename it as a future author thinks only Amazon Titan embedding model works